### PR TITLE
Make heading detection case insensitve

### PR DESCRIPTION
**Pull request by jlward**
In `get_style_dict` when we are determining if a p tag is actually a heading tag it was assuming that the value of the style would be `Heading \d` with an upper case H. However it is possible for the style to be `heading \d`. This needs to be handled

### CODE REVIEW

**Comment by winhamwr** [comment]
I'm not sure what this means. I'm guessing we're talking about [is_header](https://github.com/PolicyStat/docx2html/blob/master/docx2html/core.py#L174), but I don't see which part of that is case sensitive. Top level roman numeral detection maybe?

**Comment by winhamwr** [comment]
The test case is a bit hard to follow, but the fix looks straight-forward.

**Comment by winhamwr** [comment]
Are there any other styles that we're currently treating as case-sensitive that shouldn't be? List styles?

**Comment by jlward** [comment]
I don't think there is anything else that should be treated as case insensitive
